src/main/java/org/m14n/moonthespire/cards/EbbAndFlow.java

```diff
@@ -0,0 +1,105 @@
+package org.m14n.moonthespire.cards;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.actions.common.DiscardAction;
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.actions.common.InstantKillAction;
+import com.megacrit.cardcrawl.actions.unique.WhirlwindAction;
+import com.megacrit.cardcrawl.actions.utility.DrawPileToHandAction;
+import com.megacrit.cardcrawl.actions.utility.HandCheckAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.CardGroup;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.cards.red.Whirlwind;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.CardStrings;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.ui.panels.EnergyPanel;
+import org.apache.logging.log4j.core.lookup.AbstractConfigurationAwareLookup;
+import org.m14n.moonthespire.MoonCharacter;
+import org.m14n.moonthespire.PhaseChangeSubscriber;
+import org.m14n.moonthespire.PhaseSystem;
+
+import static org.m14n.moonthespire.Mod.cardPath;
+import static org.m14n.moonthespire.Mod.makeId;
+
+public class EbbAndFlow extends AbstractMoonCard implements PhaseChangeSubscriber {
+    public static final String ID = makeId("EbbAndFlow");
+    public static final String IMG = cardPath("moondrop.png");
+
+    private static final int COST = 2;
+    private static final CardType TYPE = CardType.ATTACK;
+    private static final CardColor COLOR = MoonCharacter.Enums.MOON_PURPLE;
+    private static final CardRarity RARITY = CardRarity.UNCOMMON;
+    private static final CardTarget TARGET = CardTarget.ENEMY;
+    private static final CardStrings cardStrings;
+
+    public EbbAndFlow() {
+        super(ID, IMG, COST, TYPE, COLOR, RARITY, TARGET);
+        PhaseSystem.subscribe(this);
```

Comment:
Style-wise I'd say move the subscription below the initialization of the members.